routers/web/repo/setting/secrets.go

```diff
@@ -27,6 +28,7 @@ type secretsCtx struct {
 	IsRepo          bool
 	IsOrg           bool
 	IsUser          bool
+	IsAdmin         bool
```

Comment:
Maybe `IsGlobal` is better than `IsAdmin`, because `IsAdmin` sounds like it will serve for admin but not others.

Reply:
Renamed and updated